packages/cli/src/commands/deploy/index.ts

```diff
@@ -15,6 +15,7 @@ import { Scope } from '../../storyblok/storyblok-client'
 export type DeployArgs = {
   skipPrompts: boolean
   dir: string
+  publish: boolean
```

Comment:
@BibiSebi is this value coming from the flag `--no-publish`? Maybe I missed it, but where the flag is `--no-publish` is giving the value to this boolean? 🤔

Reply:
Yeah its rather weird and unintuitive but when defining `-noo-publish` option with commander.js 

it will automatically translate it into `publish:false` and if you do not provide the option then it will be per default `publish:true`

Reply:
Oh, next level of a smart tool 🤯 Love it!